README.md

```diff
@@ -792,6 +792,7 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 
 * [graphql](https://github.com/tmc/graphql) - graphql parser + utilities.
 * [graphql](https://github.com/sevki/graphql) - GraphQL implementation in go.
+* [graphql](https://github.com/neelance/graphql-go) - GraphQL server with a focus on ease of use.
```

Comment:
The repo name and README say `graphql-go`, but the package name is `graphql`. Which should we use? Any preference @neelance?